puppet/modules/chassis/templates/local-config-db.php.erb

```diff
@@ -13,17 +13,17 @@ defined( 'ABSPATH' ) or define( 'ABSPATH', '<%= @wpdir %>/' );
 defined( 'WP_CONTENT_DIR' ) or define( 'WP_CONTENT_DIR', '<%= @contentdir %>' );
 
 <% if @network != false %>
+define( 'MULTISITE', true );
```

Comment:
@BronsonQuick So adding MULTISITE inside the if statement will cause user activation links to not work, so we'd need to test this before we merge it in.

Reply:
@stuartshields See: https://github.com/Chassis/Chassis/pull/629#issuecomment-494385875 currently they don't work now which is the issue I'm trying to fix.

Can you please try and test this too? Thanks dude!

Reply:
Should really read stuff before commenting 👍